### ISSUE TITLE
org.neo4j.driver.exceptions.ClientException Unknown function 'ID'

### ISSUE THREAD

**Issue by kaplan441** [neo4j/neo4j#12473]
I cloned the belowing example code.
https://github.com/neo4j-examples/spring-data-neo4j-intro-app

I started same application in my computer then it throwed an exception.

org.neo4j.driver.exceptions.ClientException Unknown function 'ID'

-Neo4j version : 3.5.16
-Spring boot example code and pom.xml dependency are same in GitHub link
-Computer language is Turkish

Please help me.

**Comment by shammakh2** [comment]
I have seen you raise the same issue in 3 different places (including neo4j community forums) . 
I have also looked at the example code and error message you provided.
It appears that it is not the 'same' application as you say it is because I have looked into it and have not found any issues with it.
You must have modified it (or perhaps used another version (which is unlikely)).
The error is telling you that there is no function "ID"
perhaps you intended to use function "id"
